src/main/java/com/gmail/goosius/siegewar/playeractions/PlunderTown.java

```diff
@@ -88,9 +100,61 @@ public static void processPlunderTownRequest(Player player,
 			if(nationOfPlunderingResident != siege.getAttacker())
 				throw new TownyException(Translation.of("msg_err_siege_war_cannot_plunder_without_victory"));
 
-			plunderTown(siege, townToBePlundered, (Nation)siege.getAttacker());
+			plunderGovernment(siege, townToBePlundered, (Nation)siege.getAttacker());
 		}
     }
+    
+    private static void plunderGovernment(Siege siege, Government loser, Government winner) {
+    	if (loser instanceof Town) {
+    		/*
+    		 * This is a town being plundered by a Nation.
+    		 */
+    		plunderTown(siege, (Town) loser, (Nation) winner);
+    	} else {
+    		/*
+    		 * This is a nation being plundered by a Town in a successful REVOLT siege.
+    		 */
+    		plunderNation(siege, (Nation) loser, (Town) winner);
+    	}
+    	
+    }
+
+    private static void plunderNation(Siege siege, Nation nation, Town town) {
+    	//Get plunder cost from the capital's size.
+    	double totalPlunderAmount =
+				SiegeWarSettings.getWarSiegeAttackerPlunderAmountPerPlot()
+				* nation.getCapital().getTownBlocks().size()
```

Comment:
There is a question @Goosius1 if the Town which revolts wins, should the plunder not be calculated off of the Capital cities' size?

Reply:
- As per convo (and previous comment also), "plunder" is defined as the plunder of a town.
- Thus the above new methods "plunderGovt" and "plunderNation" should be removed.
*note*
- Nations do occasionally suffer money loss in war, but this is done via the different "WarChest" mechanic, rather than the plunder mechanic.